### ISSUE TITLE
Crash when changing desktop background

### ISSUE THREAD

**Issue by Jude24to25** [jenius-apps/ambie#186]
Ambie crashes when:
1. The desktop background automatically changes per Windows user settings.
2. The user right-clicks the desktop background and selects "Next desktop background"

(I already mentioned this in another post, but now I'm able to replicate it again)

**Comment by dpaulino** [comment]
Wow! Ambie crashes when you change your desktop wallpaper?! that's a crazy interesting bug. I will investigate soon! Thank you for creating an issue!

**Comment by Jude24to25** [comment]
@dpaulino Thank you for the super fast response! I'm truly enjoying your app, so reporting an issue is the least I could do!  🤓👍

**Comment by dpaulino** [comment]
@Jude24to25 I am trying to reproduce the issue but it seems I can't. Can you still reproduce this with version 2.0.4?

**Comment by Jude24to25** [comment]
@dpaulino I had the app crash today twice when I did the "next background" function. However, I was not able to reproduce it after that. Normally the way I use the app is I have the app playing, another music app playing, and the Windows Dolby Atmos plugin. I have also been using a remote screen software (Luna beta), so it could be that maybe that is contributing somehow. Over the next few days, I'll see if I can figure out if there are any particular conditions that need to be present to reproduce the bug, and then I'll get back to you! 🤓👍

**Comment by Jude24to25** [comment]
@dpaulino Just as a heads up, I experienced the crash with v2.0.7.0 after doing the "next background" function. I'll keep trying to see if I can figure out the exact circumstances, though I had the same setup as described in my previous post

**Comment by dpaulino** [comment]
@Jude24to25 what OS build are you running? For example, here's my machine
![Uploading image.png…]()

**Comment by Jude24to25** [comment]
I'm on: Version 21H1 (OS Build 19043.1081)

**Comment by Jude24to25** [comment]
I can confirm the issue is still present in v2.0.8.0.
The app crashed, even when it was the only app playing, and without any Dolby Atmos sound enabled. I restarted the computer, and the app crashed again when I did "Next Background". The problem is that it's not consistent. Sometimes it crashes when I do "Next Background", and sometimes it does not.

**Comment by dpaulino** [comment]
@Jude24to25 are you still having issues? If so, would be great to have a video recording!

**Comment by Jude24to25** [comment]
@dpaulino Yeah, I'm still experiencing the issue. However, I can't seem to scientifically reproduce it. I tried recording it just a moment ago, but the issue wasn't happening. And then I stopped trying to record it, and then it happened, LOL. Also, I should mention that I've had the issue on a Windows 11 computer; however, I also cannot scientifically reproduce it. Is there any bug-reporting process that we can implement that will collect crash data and send it?
